frontend/src/concepts/pipelines/content/tables/pipelineRun/PipelineRunTableRow.tsx

```diff
@@ -47,7 +47,9 @@ const PipelineRunTableRow: React.FC<PipelineRunTableRowProps> = ({
           <TableRowTitleDescription
             title={
               <Link to={`/pipelineRuns/${namespace}/pipelineRun/view/${run.id}`}>
-                {data ? `Run of ${data.name}` : run.name}
+                <TableText wrapModifier="truncate">
+                  {data ? `Run of ${data.name}` : run.name}
+                </TableText>
```

Comment:
does `<Td modifier="truncate">` work here? Likely not as i see lots of nesting of other components. Seems like PF has too many options to do the same thing...